client/src/tutorials/BattleCoins/pages/2.md

```diff
@@ -0,0 +1,21 @@
+# Getting Started
+
+This program will be using the `anchor_spl` and `mpl_token_metadata` crates to help us interact with the Token program and the Token Metadata program.
```

Comment:
Could you add links to those crates' documentation so that it makes it easier for people to check the types etc?

```suggestion
This program will be using the [`anchor_spl`](https://docs.rs/anchor-spl/latest/anchor_spl/index.html) and [`mpl_token_metadata`](https://docs.rs/mpl-token-metadata/latest/mpl_token_metadata/) crates to help us interact with the Token program and the Token Metadata program.
```